javascript/javascript2/week3/Game/game.js

```diff
@@ -0,0 +1,61 @@
+const input = document.getElementById("input");
+const startButton = document.getElementById("startbtn");
+const restartButton = document.getElementById("restartbtn");
+const sKey = document.getElementById("keys");
+const lKey = document.getElementById("keyl");
+const getResult = document.getElementById("result")
+const getAlert = document.getElementById("alert")
+let sCount = 0;
+let lCount = 0;
+
+const startGame = () => {
+
+  if (sCount == lCount) {
+    getResult.innerHTML = "Game Draw!!!";
+  }
+  if (sCount > lCount) {
+    getResult.innerHTML = "S is the Winner !!!";
+  }
+  if (lCount > sCount) {
+    getResult.innerHTML = "L is the Winner";
+  }
+};
+startButton.addEventListener("click", gameStart);
+
+function gameStart() {
+  let getTime = input.value;
+  if (getTime == "") {
+    getAlert.innerHTML = "Please enter time!!";
```

Comment:
You will not be able to display this message, first `getAlert` will get `Please enter time!!` then when it comes on line 34  `getAlert` will get `Game Over!!`. Means it will be always `Game Over!!`. to fix this, I suggest to use `else` like this:
```
if (getTime === '') {
		getAlert.innerHTML = 'Please enter time!!';
		document.removeEventListener('keydown', keyPressFunc);
	}else{
	setTimeout(() => {
		document.removeEventListener('keydown', keyPressFunc);
		getAlert.innerHTML = 'Game Over!!';
		startGame();
	}, getTime * 1000);
    }
```

Reply:
Thanks for pointing out .